### ISSUE TITLE
Error while building colmap with vcpkg, any alternative with prebuilt colmap?

### ISSUE THREAD

**Issue by mpizenberg** [nerfstudio-project/nerfstudio#1002]
I tried installing colmap with vcpkg as suggested in the documentation `vcpkg install colmap[cuda]:x64-windows`

I faced an error which is described in the following issue: https://github.com/microsoft/vcpkg/issues/27956

So meanwhile, I already have a prebuilt colmap installed, and I have the `COLMAP.bat` in my path. Yet when I’m running `ns-process-data` it says that COLMAP is not found.

Do you have any info if there is a workaround? Especially if I can avoid having to recompile all I need myself.

OS: Windows 10
nerfstudio: installed with pip in a conda environment.

**Comment by devernay** [comment]
did you try
```
conda install -y -c conda-forge colmap
```
looks like it's available for win-64 (and ffmpeg too if you need it)

**Comment by mpizenberg** [comment]
I so that in the google collab setup so I tried just before leaving work and the colmap I got seemed to not be working (returned immediately without running anything). I'll dig deeper in the coming days, but not tomorrow as I've other things on my plate right now and was exploring nerfstudio while waiting on other things at work. Anyway, I'll definitely persevere in the coming days / weeks and report here solutions or issues.

**Comment by mpizenberg** [comment]
So colmap with vcpkg still does not install for me, and unfortunately colmap installed via conda does not work.
```
conda install -c conda-forge colmap
```
After installation, I have a colmap executable available, but it simply returns without doing anything. Is this something specific to the install on windows, or specific to me? Are there other people on windows where the install via conda (4.12.0) works?

**Comment by mpizenberg** [comment]
I got an update from vcpkg. Apparently there is a known (unsolved) bug with Visual Studio 2022 preventing from compiling colmap with vcpkg: https://developercommunity.visualstudio.com/t/Internal-compiler-error-when-install-col/10035653

So you might want to warn users about this issue for windows in the guide section about installing colmap https://docs.nerf.studio/en/latest/quickstart/custom_dataset.html#installing-colmap

**Comment by mpizenberg** [comment]
I'm not sure what I did wrong the first time around when trying to use colmap with the prebuilt archive. I retested everything and now it seems to be working. So as a summary for how to get colmap working with nerfstudio on windows:

- OK: extract prebuilt colmap from [latest release](https://github.com/colmap/colmap/releases) and add to the PATH the directory containing the `COLMAP.bat` file
- NOT ok: install via vcpkg. There is a visual studio 2022 bug.
- NOT ok: install via conda. The executable returns immediately.

After trying many things, the conda behavior is the same as if trying to execute the `bin/colmap.exe` file inside the colmap archive. So maybe I'm wrong, but it could be a hint that the conda install places the `bin/colmap.exe` in our path instead of the bat file.
